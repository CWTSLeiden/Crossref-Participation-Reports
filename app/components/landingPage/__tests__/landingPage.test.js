import React from 'react';
import Enzyme, { shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import toJson from 'enzyme-to-json';
import sinon from 'sinon'

import LandingPage from '../landingPage'




const shallowComponent = shallow(<LandingPage/>)


test('shallow snapshot', () => {
  expect(toJson(shallowComponent)).toMatchSnapshot()
})