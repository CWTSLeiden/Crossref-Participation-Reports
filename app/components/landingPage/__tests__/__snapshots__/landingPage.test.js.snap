// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow snapshot 1`] = `
<div
  className="landingPage"
>
  <div
    className="topStripe"
  >
    <div
      className="searchContainer"
    >
      <div
        className="searchTitle"
      >
        Find a Publisher
        <p
          className="bannerText"
        >
          How good is your metadata?
        </p>
      </div>
      <Search />
    </div>
  </div>
  <div
    className="content"
  >
    <div
      className="greyBar"
    >
      <img
        src="/participationreports/images/light-bulb.svg"
      />
      <div
        className="greyBarContent"
      >
        <p
          className="smallText"
        >
          Comprehensive metadata makes publications discoverable.
        </p>
        <p
          className="bigText"
        >
          Make sure your content can be found.
        </p>
        <div
          className="button"
        >
          Learn More
        </div>
      </div>
    </div>
    <div
      className="testimonialContainer"
    >
      <div
        className="testimonial"
      >
        <p
          className="testimonialBody"
        >
          “Our team deposit a lot of metadata. With metadata Reports we now have a central dashboard for viewing all our depositing activity.“
        </p>
        <p
          className="testimonialAuthor"
        >
          Laura Knight from SciMed Pub
        </p>
      </div>
      <div
        className="testimonial"
      >
        <p
          className="testimonialBody"
        >
          “Metadata Reports meets our needs perfectly. For visualising and sharing metadata activity to all our organisation.”
        </p>
        <p
          className="testimonialAuthor"
        >
          Seth Jones from HyperNitro Pub
        </p>
      </div>
      <div
        className="testimonial"
      >
        <p
          className="testimonialBody"
        >
          “Easy to use and super helpful, exactly what our team needed.”
        </p>
        <pre
          className="testimonialAuthor"
        >
          Jimmy Bone
from Ultra LightSpeed Publications
        </pre>
      </div>
    </div>
  </div>
</div>
`;
